elementor/elementor
# Fix for #8084 - WP image alignment overrides

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fixes margin overrides affecting default WP alignment classes

## Description


* Fixes margin overrides affecting default WP alignment classes

## Test instructions
This PR can be tested by following these steps:

* Create a standard post with an image set to align-left, align-right, align-center
* Preview and check that the margin is preserved

## Quality assurance


Fixes #8084
